### ISSUE TITLE
responseType is not passed to the google API call

### ISSUE THREAD

**Issue by farhantahir** [anthonyjgrove/react-google-login#337]
`<GoogleLogin
                clientId="MY_CLIENT_ID"
                buttonText="Continue with Google"
                uxMode="redirect"
                onFailure={(res) => console.log(res)}
                responseType="token"
              />`

I need to get "accessToken" and as per docs of the plugin.

> Can be either space-delimited 'id_token', to retrieve an ID Token + 'permission' (or 'token'), to retrieve an Access Token, or 'code', to retrieve an Authorization Code.

But this is not the case.

**Comment by mtbtalha** [comment]
@farhantahir  Did you find any solution to this?

**Comment by danbiagini** [comment]
This looks like a trivial fix unless I'm missing something... 

```diff --git a/src/use-google-login.js b/src/use-google-login.js
index 111038e..50fd592 100644
--- a/src/use-google-login.js
+++ b/src/use-google-login.js
@@ -83,9 +83,12 @@ const useGoogleLogin = ({
         ux_mode: uxMode,
         redirect_uri: redirectUri,
         scope,
-        access_type: accessType
+        access_type: accessType,
+        response_type: responseType
       }

+      // eslint-disable-next-line no-console
+      console.log(`responseType:${responseType}`)
       if (responseType === 'code') {
         params.access_type = 'offline'
       }
```

I am trying to test it locally but having an issue getting a local react-google-login component to work in my app.  If anyone has any ideas I'd be happy to test this out and make a PR..

```
Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
```

**Comment by farhantahir** [comment]
@mtbtalha No, I'm not actually rather that, I was understanding it differently, accessToken can only be get via back-end calls I guess, thus what approach I used was get the received id_token from this API, send it our back-end and then verify it using google scripts.

For more details: https://developers.google.com/identity/sign-in/web/backend-auth#verify-the-integrity-of-the-id-token

**Comment by anze555** [comment]
@danbiagini I can confirm that your fix above works, thank you very much! Much appreciated.

To use (or test) this fix:
1) clone this repository and add the missing line to `use-google-login.js`, as per [this comment](https://github.com/anthonyjgrove/react-google-login/issues/337#issuecomment-653336044)
2) `npm run bundle`
3) copy the resulting file (`dist/google-login.js`) to _**your**_ React app and put it in `src/` folder
4) wherever you use `react-google-login`, instead of importing like this:
    ```
    import { GoogleLogin } from "react-google-login";
    ```
    use:
    ```
    import { GoogleLogin } from "../../google-login";
    ```
    (relative paths need to be correct, of course)
5) remove the original version of `react-google-login`: `npm uninstall react-google-login`

Hope it helps someone.
